src/utils/publish-lock.js

```diff
@@ -1,13 +0,0 @@
-// @flow
```

Comment:
This file didn't seem to be used for anything... 🤷‍♀️

Reply:
.... ... Hmmm.. I think at some point I must have copy + pasted it in VSCode and not noticed it. Its a copy of the file from commands (not sure they /should/ live there either, but thats a different point)